Summary: Don't strip here.
Contributor: Decklin Foster <decklin@red-bean.com>

Index: netcat-1.10/Makefile
===================================================================
--- netcat-1.10.orig/Makefile	2006-03-24 12:25:49.000000000 -0500
+++ netcat-1.10/Makefile	2006-03-24 12:26:01.000000000 -0500
@@ -14,8 +14,8 @@
 XLIBS =		# xtra libs if necessary?
 # -Bstatic for sunos,  -static for gcc, etc.  You want this, trust me.
 STATIC =
-CC = cc $(CFLAGS)
-LD = $(CC) -s	# linker; defaults to stripped executables
+CC = gcc $(CFLAGS)
+LD = $(CC)	# linker; defaults to unstripped executables
 o = o		# object extension
 
 ALL = nc
